photon-core/src/main/java/org/photonvision/vision/processes/VisionModule.java

```diff
@@ -395,6 +395,7 @@ public void startCalibration(UICalibrationData data) {
         settings.cameraAutoExposure = true;
 
         setPipeline(PipelineManager.CAL_3D_INDEX);
+        pipelineManager.calibration3dPipeline.broadcastState();
```

Comment:
I guess making this public isn't the worst hack. Reaching into pipeline manager from VMM is precedented, right?